index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link href="styles.css" rel="stylesheet"><title>Hello World</title>
+    </head>
+    <body>
+        <h1 class="header">Hello Microverse!</h1>
+    </body>
+</html>
```

Comment:
- [ ] This is causing linters error, please add  linters to your project to fix the issue before send a PR for review [ HTML/CSS linters link](https://github.com/microverseinc/linters-config)